playbooks/roles/airship-configure-worker/tasks/enroll-nodes.yml

```diff
@@ -47,8 +47,21 @@
   tags:
     - skip_ansible_lint
 
-- name: Enroll node for Airship Openstack compute plane
-  command: "kubectl label nodes {{ item }} openstack-compute-node=enabled openvswitch=enabled"
+- name: Enroll node for Airship Openstack compute plane (compute label)
+  command: "kubectl label nodes {{ item }} openstack-compute-node=enabled"
```

Comment:
nope, compute then openvswitch, order should not matter

Reply:
does the order matters in any way?

Reply:
ah yeash, so thanks to the check already in there it wont apply it a second time.

When converting the worker-2 into a control node as well, it already gets the openvswitch label so this was skipping and the compute label was missing, thus the need to split it into 2 tasks for the labels.

Reply:
indeed, this should be reworked somehow, but with the number of different node groups and the same labels for different groups it may be a bit ugly to do so.